src/Sentinel.php

```diff
@@ -629,6 +629,19 @@ public function removeCheckpoint($key)
     }
 
     /**
+     * Removes the chosen checkpoints
```

Comment:
"short descriptions should end in either a full stop, exclamation mark, or question mark"

`* Removes the chosen checkpoints.`